### ISSUE TITLE
img and vid textures can get swapped with eachother in some circumstances

### ISSUE THREAD

**Issue by dktr0** [dktr0/Punctual#8]

**Comment by cleary** [comment]
Was just about to report this:

```
i1 << img "https://img.grbt.com.au/scream.jpg";
v1 << vid "https://upload.wikimedia.org/wikipedia/commons/2/23/Spider_web_building.webm";
i1 >> hsv;
```
This plays the video, instead of the image

**Comment by dktr0** [comment]
I believe this was resolved with version 0.4.4.5.
